### PR TITLE
Add android_do.sh

### PR DESCRIPTION
**Pull request by sssemil**

### CODE REVIEW

File: android_do.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+ANDROID_NDK_PATH=/home/emil/dev/android-ndk-r10d/
```
Comment (cjdelisle) [inline android_do.sh]: bzzzzt not eveyone is named emil :)

Comment (sssemil) [inline android_do.sh]: Ops, should I leave it empty?

**Comment by cjdelisle** [comment]
If it doesn't work out-of-box then it doesn't make sense to put it in the main directory, if there is a way to detect the location of the ndk (and print a message explaining how to install it if it's missing) then that would be cool. Barring that it would be better to put it in /contrib/ and include some documentation on how to use it.

**Comment by sssemil** [comment]
I think it's better to think a bit more. Thank you fo your help.

**Comment by lgierth** [comment]
@sssemil also, take a look at contrib/android/, it probably does part of what you want to achieve

**Comment by sssemil** [comment]
Thanks, here is new pull request https://github.com/cjdelisle/cjdns/pull/814

**Pull request closed without merging**